<ui version="4.0" >
 <class>EditStyleBase</class>
 <widget class="QDialog" name="EditStyleBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MuseScore: Edit Style</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonApply" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QListWidget" name="pageList" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>4</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentRow" >
      <number>-1</number>
     </property>
     <item>
      <property name="text" >
       <string>Page</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Measure</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Numbers</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="page1" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="showPageNumber" >
         <property name="title" >
          <string>Page Numbers</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <widget class="QCheckBox" name="showFirstPageNumber" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>30</y>
            <width>100</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text" >
           <string>show first</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="showOddEvenPageNumber" >
          <property name="geometry" >
           <rect>
            <x>130</x>
            <y>30</y>
            <width>100</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text" >
           <string>odd/even</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="showMeasureNumber" >
         <property name="title" >
          <string>Measure Numbers</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="1" >
           <widget class="QCheckBox" name="showAllStaffsMeasureNumber" >
            <property name="text" >
             <string>all staffs</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="showEverySystemMeasureNumber" >
            <property name="text" >
             <string>every system</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QCheckBox" name="showFirstMeasureNumber" >
            <property name="text" >
             <string>show first</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QCheckBox" name="showIntervalMeasureNumber" >
            <property name="text" >
             <string>interval</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QSpinBox" name="intervalMeasureNumber" >
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="value" >
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page2" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="4" column="2" >
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>spaces</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>spaces</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>spaces</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>spaces</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>spaces</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>staff upper border</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>staff lower border</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>staff distance</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>akkolade distance</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" >
        <widget class="QDoubleSpinBox" name="staffLowerBorder" />
       </item>
       <item row="2" column="1" >
        <widget class="QDoubleSpinBox" name="staffDistance" />
       </item>
       <item row="3" column="1" >
        <widget class="QDoubleSpinBox" name="akkoladeDistance" />
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>system distance</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QDoubleSpinBox" name="systemDistance" />
       </item>
       <item row="0" column="1" >
        <widget class="QDoubleSpinBox" name="staffUpperBorder" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page3" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="3" column="2" >
        <widget class="QLabel" name="label_17" >
         <property name="text" >
          <string>spaces</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QLabel" name="label_16" >
         <property name="text" >
          <string>spaces</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="label_15" >
         <property name="text" >
          <string>spaces</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_14" >
         <property name="text" >
          <string>min note distance</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_13" >
         <property name="text" >
          <string>min. right margin</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_12" >
         <property name="text" >
          <string>left margin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_11" >
         <property name="text" >
          <string>spacing (1=tight)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QDoubleSpinBox" name="minNoteDistance" />
       </item>
       <item row="2" column="1" >
        <widget class="QDoubleSpinBox" name="noteBarDistance" />
       </item>
       <item row="1" column="1" >
        <widget class="QDoubleSpinBox" name="barNoteDistance" />
       </item>
       <item row="0" column="1" >
        <widget class="QDoubleSpinBox" name="measureSpacing" />
       </item>
       <item row="4" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>EditStyleBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>318</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>EditStyleBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>409</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageList</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>42</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel" >
     <x>395</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
